### DISCUSSION TITLE
躋身獨角獸的矽谷新 AI 公司--Cresta

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS4719-C/Python-Community-GS4719-C#537] [General]
**Cresta的創立**
Cresta是史丹佛系創業公司，三位聯合創始人為史丹佛人工智慧實驗室博士生Zayd Enam和施天麟（Tim Shi），以及史丹佛大學教授、自動駕駛之父、Udacity創始人Sebastian Thrun。

**Cresta在做什麼？**
2017年籌備成立開始前Cresta發現，基於對話的客戶服務和銷售業務在今天的消費經濟當中十分重要，但卻沒有得到與其重要性所匹配的關注。
目前全球從事客服行業的人員總量達到約1700萬人，每天都有大量的客戶服務人員在跟使用者進行電話、文字、影片溝通，但該行業普遍存在著缺乏訓練、人手不足的問題。
Cresta祕密研發了一套完整的人工智慧系統，旨在終結這些困擾客戶服務和銷售行業已久的現狀，從而幫助企業提高整個品牌的客戶體驗。 截至目前，Cresta主要為企業提供面向客服人員（For agents）、面向管理者（For managers）、面向商業（For Leadership）三大類智慧客服解決方案。

**三大面向For agents,For managers,For Leadership**
For agents:通過實時和AI驅動的協助輔導，能自動執行識別情緒、智慧反應、知識庫查詢和自動記錄等操作。
For managers:Cresta 會跟蹤座席表現，通過分析客服每一次對話來為管理人員提供端到端的指導，使之能更有針對性地提高客服質量。
For Leadership:基於整個客服中心表現和回饋，提供可執行建議，包括有效辨識話題、情緒和競爭問題，幫助企業做到更智慧靈活的客戶體驗。

Cresta能發現各個企業最需要且最不足的地方進而幫助他們，利用AI技術提供最好的服務真的很聰明。
我也很喜歡Cresta的理念：AI 不是替代人工，而是成為更好的工具，幫助人類實現工作效率的100倍提高。

資料來源：https://technews.tw/2022/03/29/tiger-global-backs-crestas-real-time-intelligence-as-company-value-quadruples/
[ 以下附上Cresta這間潛力無窮AI公司的官網](https://cresta.com)

**Comment by chia-kai-chang** [comment]
除非文章是用自己的話語，重新詮釋說明，否則請附上資料來源

**Reply by liangyinglly** [reply]
已更新

**Reply by chia-kai-chang** [reply]
好唷~
